docs/user/tutorial/30-structure-content-modes.rst

```diff
@@ -43,19 +44,20 @@ Add a second plugin
 
 Let's add another plugin, containing a map.
 
-.. todo:: add the "Add plugin" icon as add-plugin-icon.png
+.. todo:: Daniele please check add the "Add plugin" icon as add-plugin-icon.png
 
-.. |add-plugin-icon| image:: /user/tutorial/images/add-plugin-icon.png
+.. image:: /user/tutorial/images/add-plugin-icon.png
    :alt: 'add plugin'
+   :width: 300
 
 #.  Select the **Add plugin** icon |add-plugin-icon|.
 #.  Choose *Google Map* from the list of available plugin types.
 
     .. todo:: replace the image, with one showing Google Map in the list
 
-    .. image:: /user/tutorial/images/text-plugin.png
+    .. image:: /user/tutorial/images/google-map-plugin.png
          :alt: the list of plugin types
-         :width: 165
+         :width: 200
```

Comment:
Looks OK, but do you think that a screenshot with more context would be more useful to the user? I am not sure.